.github/scripts/run-fips-it.sh

```diff
@@ -15,4 +15,6 @@ TESTS=`testsuite/integration-arquillian/tests/base/testsuites/suite.sh fips`
 echo "Tests: $TESTS"
 export JAVA_HOME=/etc/alternatives/java_sdk_17
 set -o pipefail
-./mvnw test -Dsurefire.rerunFailingTestsCount=$SUREFIRE_RERUN_FAILING_COUNT -nsu -B -Pauth-server-quarkus,auth-server-fips140-2 -Dcom.redhat.fips=false $STRICT_OPTIONS -Dtest=$TESTS -pl testsuite/integration-arquillian/tests/base | misc/log/trimmer.sh
+
+# Profile app-server-wildfly needs to be explicitly set for FIPS tests
```

Comment:
The `app-server-wildfly` was explicitly set in this script in order to execute all tests properly, even though the app server is the default one in our testsuite. Otherwise, Arquillian says there's some problem with finding app-server-wildlfy. This behavior is probably caused by a different execution of these tests with additional maven profiles. 

@vmuzikar The investigation of how to remove the explicit specification of the server in this script can be done in a follow-up task, ok? It's only testsuite-related thing and doesn't affect FIPS itself at all. I want to avoid blocking this PR and Friday's Quarkus 3 code freeze.